File: 58-length-of-last-word/58-length-of-last-word.c
int lengthOfLastWord(char * s){
    int c = 0, len = strlen(s);
    
    while(len >= 0)
    {
        if(s[len-1] == ' ' || s[len-1] == '\t')
            len--;
        else
        {
            while((len-1 >= 0) && s[len-1] != ' ')
            {
                c++;
                len--;
            }
            return c;
        }
    }
    return 0;
}

File: 58-length-of-last-word/58-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int last = s.length()-1;
        int c = 0;
        
        while(last >= 0 && s.charAt(last) == ' ') 
            last--;
        
        for(; last >= 0; last--){
            if(s.charAt(last) != ' ') c++;
            else break;
        }
        
        return c;
    }
}